avm/res/sql/server/README.md

```diff
@@ -1139,6 +1139,7 @@ module server 'br/public:avm/res/sql/server:<version>' = {
 | [`enableTelemetry`](#parameter-enabletelemetry) | bool | Enable/Disable usage telemetry for module. |
 | [`encryptionProtectorObj`](#parameter-encryptionprotectorobj) | object | The encryption protection configuration. |
 | [`firewallRules`](#parameter-firewallrules) | array | The firewall rules to create in the server. |
+| [`isIPv6Enabled`](#parameter-isipv6enabled) | string | Whether or not to enable IPv6 support for this server. |
```

Comment:
Hey @peterbud thanks for this contribution, looks great!
Since new functionality is introduced I'd suggest to also update the minor version in the corresponding version.json file

Reply:
OK, I'll do that. I'm in conversation with @krbar on how to make the tests run more smoothly: currently the max tests cases are hitting regional limitations (zone redundancy). I hope that we can sort out in the coming days

Reply:
Also, an attached pipeline badge to validate that the static & deployment tests passed would be great :)

Reply:
> Hey @peterbud thanks for this contribution, looks great! Since new functionality is introduced I'd suggest to also update the minor version in the corresponding version.json file

I have bumped the version.

Also after discussing with @krbar, I have pinned the location for `max` and `waf` test to `uksouth`, to avoid issues with limited zone redundancy in some regions.

> Also, an attached pipeline badge to validate that the static & deployment tests passed would be great :) 

I can add that unfortunately only when my subscription will be re-enabled on the 10th Oct. One of the max test deployment was not deleted, and by the time I noticed I already reached the limit of my subscription.

Reply:
@krbar @AlexanderSehr in this case do you think we can have an exception and have the PR merged? Personally I'd go for it having a look at its run completion

Reply:
@peterbud, regarding the test resources, please enure you ran the 'toggle workflows' workflow once in your fork (it's called '.Platform: Toggle Workflows' or similar). It will disable all workflows and hence save you from a large amount of module workflows running due to their trigger on main, ultimately saving you costs. You can then enable the workflows you want to test individually (like the one for SQL-Sever).

Reply:
All workflows were/are disabled, it was my manual test which left this resource in my subscription, so it's my fault at the end.

Reply:
> @krbar @AlexanderSehr in this case do you think we can have an exception and have the PR merged? Personally I'd go for it having a look at its run completion

@eriqua Pls see the workflow run badge from @krbar, I have included that in the top entry.